Drupal 10 compatability

mikzalais opened the pull request.

mikzalais committed:
Update SessionManagerUserLoginEvent.php

D10 compat

mikzalais committed:
Update SessionManagerUserLogoutEvent.php

D10 compat

mikzalais committed:
Update SessionManagerEventSubscriber.php

D10 compat

mikzalais committed:
Merge branch 'wunderio:8.x-1.x' into 8.x-1.x

becizzz reviewed (approved):
LGTM

becizzz commented:
@mikzalais Is it so that with these changes the module is no more compatible with v8 and v9? If so then the info.yml also needs to be changed accordingly.

mikzalais commented:
Ok, makes sense.

Sveicieni! :)

mikzalais committed:
Update session_manager.info.yml

mikzalais committed:
Update session_manager.module

D10

mikzalais committed:
Update SessionManagerService.php

D10 compat.

Issue with session_manager not getting session id:
https://www.drupal.org/node/3006306
https://www.drupal.org/project/cas_server/issues/3465130

mikzalais committed:
Drupal 11
